awx/main/dispatch/control.py

```diff
@@ -0,0 +1,55 @@
+import logging
```

Comment:
This module contains code used to send RPC commands to the dispatcher.  On my first pass, I've implemented two:

`awx-manage run_dispatcher --status`; this sends a message to the dispatcher that causes it to gather status information on the worker pool (how many processes, what they're working on) and report that data *back* to the user over AMQP.

Reply:
you said you implemented two - was the other one the command to recycle workers after jobs finished?

Reply:
(updated my comment)